index.html

```diff
@@ -407,56 +399,39 @@ <h3>
-  </aside>
-  <pre class="idl">
-    enum PressureFactor { "thermal", "power-supply" };
-  </pre>
+<section> <h2>Contributing Factors and Hysteresis</h2>
   <p>
-    The <dfn>PressureFactor</dfn> enum represents the [=contributing factors=]:
-  </p>
-  <p>
-    <ul class="pressure-factors">
-      <li>
-        {{PressureFactor/"thermal"}}: The source is under elevated thermal conditions and the system is
-        unable to keep temperature below safe limits at peak power.
-      </li>
-      <li>
-        {{PressureFactor/"power-supply"}}: The power supply (incl. battery capacity) is too weak to run at peak power,
-        or artificially limited by a system or user initiated power mode to conserve power or
-        avoid fan noise.
-      </li>
-    </ul>
+    Contributing factors represents the underlying hardware metrics contributing to the [=current pressure state=]
+    and can be [=implementation-defined=].
   </p>
   <p>
     The <dfn>change in contributing factors is substantial</dfn> steps are as follows:
     <ol>
       <li>
         If [=implementation-defined=] low-level hardware metrics that contribute to the
-        system performance and [=current pressure state=] drop below or exceed an
-        [=implementation-defined=] threshold for the [=current pressure state=], return true.
+        [=current pressure state=] drop below or exceed an, per metric,
+        [=implementation-defined=] threshold
+        for the [=current pressure state=], return true.
       </li>
       <li>
         Return false.
       </li>
     </ol>
     <aside class="note">
-      The [=change in contributing factors is substantial=] algorithm allows user agents to avoid
-      flip-flopping between two states in certain circumstances. For example, a state might otherwise
-      change too rapidly in response to a certain system metric that fluctuates around a boundary
-      condition that triggers a state change. This specification does not define the precise algorithm
-      to allow implementations optimize this algorithm to match the underlying hardware platform's behavior.
-      One possible implementation of this algorithm is to use a
-      <a href="https://en.wikipedia.org/wiki/Preisach_model_of_hysteresis#Nonideal_relay">nonideal relay</a>
-      as a model and identify appropriate lower threshold &#945; and upper threshold &#946; for each
-      [=pressure state=] taking special characteristics of each [=contributing factor=] into consideration.
+      <p>
+        The [=change in contributing factors is substantial=] algorithm is based on the idea of hysteresis and
+        allows [=user agents=] to avoid flip-flopping between states in certain circumstances. For example, a
+        state might otherwise change too rapidly in response to a certain system metric that fluctuates around
+        a boundary condition that triggers a state change.
+      </p>
```

Comment:
I think the section above shouldn't be talking about hysteresis but simply about algorithm that filters high frequency changes or jittering between values.

Algorithms could be averaging (low pass filtering) for high frequency 
or hysterisis for jittering.

Then the next section as you did should give some examples, but they are not a part of the specs.